src/components/BuildConfigCreateEditPage/BuildConfigCreateEditPage.tsx

```diff
@@ -253,14 +258,62 @@ export const BuildConfigCreateEditPage = ({ isEditPage = false }: IBuildConfigCr
   };
 
   const submitEdit = (data: IFieldValues) => {
```

Comment:
Once form is submitted, there is high amount of cancelled requests, the reasons needs to be investigated and new Jira needs to be created if it's a new bug.

Reply:
The has already been cretaed before: NCL-8349.